deploy/chart/templates/07-subscription.crd.yaml

```diff
@@ -29,11 +29,16 @@ spec:
           required:
           - source
           - name
+          - sourceNamespace
```

Comment:
This shouldn't be required, right?

This will cause existing Subscriptions to be invalid, and I think we can assume that the sourceNamespace is the catalogNamespace if not specified.